crates/holochain/src/core/sys_validate/tests.rs

```diff
@@ -298,8 +301,14 @@ async fn check_previous_action() {
         cascade
             .expect_retrieve_action()
             .times(2)
-            // Doesn't matter what we return, the action should be rejected before deps are checked.
-            .returning(|_, _| async move { Ok(None) }.boxed());
+            // Doesn't matter what we return, the action should be rejected before deps are checked,
```

Comment:
This calls `sys_validate_record` which calls the inline validation function, so regardless of authority every action is checked for agent validity.